itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -350,6 +353,177 @@ public void testAcidTablesCreateTableIncremental() throws Throwable {
       .verifyResults(new String[] {"11"});
   }
 
+  @Test
+  public void testFailoverWithNoOpenTxns() throws Throwable {
+    WarehouseInstance.Tuple dumpData = null;
+
+    dumpData = primary.run("use " + primaryDbName)
+            .run("create table t1 (id int) clustered by(id) into 3 buckets stored as orc " +
+                    "tblproperties (\"transactional\"=\"true\")")
+            .run("create table t2 (rank int) partitioned by (name string) tblproperties(\"transactional\"=\"true\", " +
+                    "\"transactional_properties\"=\"insert_only\")")
+            .dump(primaryDbName);
+
```

Comment:
Pass the failover start config to true for bootstrap and verify the file does not exist

Reply:
Done